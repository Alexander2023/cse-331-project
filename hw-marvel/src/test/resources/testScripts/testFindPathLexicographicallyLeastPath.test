# Tests that BFS chooses the lexicographically least path when two paths of same length

CreateGraph g

AddNode g c1
AddNode g c2
AddNode g c3
AddNode g c4

AddEdge g c1 c2 b1
AddEdge g c2 c4 b2
AddEdge g c1 c3 b3
AddEdge g c3 c4 b4

FindPath g c1 c4