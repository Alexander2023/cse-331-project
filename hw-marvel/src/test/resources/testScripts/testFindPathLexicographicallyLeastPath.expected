# Tests that BFS chooses the lexicographically least path when two paths of same length

created graph g

added node c1 to g
added node c2 to g
added node c3 to g
added node c4 to g

added edge b1 from c1 to c2 in g
added edge b2 from c2 to c4 in g
added edge b3 from c1 to c3 in g
added edge b4 from c3 to c4 in g

path from c1 to c4:
c1 to c2 via b1
c2 to c4 via b2